src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -388,4 +388,17 @@ export class MatCheckbox
       nativeCheckbox.nativeElement.indeterminate = value;
     }
   }
+
+  /**
+   *  Prevent click events that come from the `<label/>` element from bubbling. This prevents the
+   *  click handler on the host from triggering twice when clicking on the `<label/>` element. After
+   *  the click event on the `<label/>` propagates, the browsers dispatches click on the associated
+   *  `<input/>`. By preventing clicks on the label by bubbling, we ensure only one click event
+   *  bubbles when the label is clicked.
+   **/
+  protected preventBubblingFromLabel(event: MouseEvent) {
```

Comment:
This PR is targeting `patch`, but the code that allows for protected properties to be used in templates is only in Angular v14. Either this should be changed to `public` or the PR should target `rc`.

Reply:
fixed